corpus/transform.py

```diff
@@ -437,10 +437,6 @@ def run(self):
 class ShiftCorpusSegmentStartJob(Job):
     """
     Shifts the start time of a corpus to change the fft window offset
-
-    :param Path corpus_file: path to a bliss corpus file
-    :param str corpus_name: name of the new corpus
-    :param float shift: shift in seconds
```

Comment:
was this supposed to be removed or moved to `__init__`?

Reply:
Ah yes, I will fix this. But anyway I will have to do a second pass of refactoring where I fix all the other issues, I wanted to start with the parameter names first.